drivers/nvme/host/core.c

```diff
@@ -3116,8 +3116,12 @@ static int nvme_init_non_mdts_limits(struct nvme_ctrl *ctrl)
 	else
 		ctrl->max_zeroes_sectors = 0;
 
+	/*
+	 * NVME_ID_CNS_CS_CTRL is supported from
+	 * v2.0.0 onwards.
+	 */
```

Comment:
Cosmetics, but why did you need to wrap this? It would perfectly fit in 80 columns.  I already have two 40" monitors I can't go much bigger to fit more in my screen. ;)

Reply:
Thanks, updated.